timemachine/ff/handlers/nonbonded.py

```diff
@@ -534,8 +516,12 @@ def static_parameterize(params, smirks, mol):
             SMIRKS patterns matching bonds, to be parsed using OpenEye Toolkits
         mol: Chem.ROMol
             molecule to be parameterized.
-
+        validate: bool
+            check params, smirks
         """
+        if validate:
```

Comment:
After talking offline about how this is intended to verify the FF and not the molecule's parameterization. What about moving this validation to the Forcefield class? It seems like longer term we will want to do more validation of the forcefield parameters as we train them.

Reply:
Hmm, it might be more appropriate for this to be in the Forcefield constructor at this branch: https://github.com/proteneer/timemachine/blob/776873a013e65f8d825570312d0fa87d3a438522/timemachine/ff/__init__.py#L71-L73

Based on discussion with @jkausrelay yesterday, the initial thought was to put this check in the core function that gets called by everything else (`static_parameterize`, which is called by `partial_parameterize`, which is called by `parameterize`, ...), so it would cover the most use cases (such as training, where we don't necessarily create an instance of the Forcefield class for every optimizer step).

I'm agnostic about where the check should go, as long as it surfaces an error/warning before applying invalid parameters to a molecule.